engine/battle/ai/items.asm

```diff
@@ -725,10 +725,14 @@ AI_HealStatus:
 	xor a
 	ld [hl], a
 	ld [wEnemyMonStatus], a
-	; Bug: this should reset SUBSTATUS_NIGHTMARE too
-	; Uncomment the lines below to fix
+	; Bug: this should reset SUBSTATUS_NIGHTMARE
+	; Uncomment the 2 lines below to fix
 	; ld hl, wEnemySubStatus1
 	; res SUBSTATUS_NIGHTMARE, [hl]
+	; This should also reset SUBSTATUS_CONFUSED
```

Comment:
"Bug: This should reset SUBSTATUS_CONFUSED"

Reply:
Fixed already.